README.md

```diff
@@ -1,85 +1,100 @@
-# BayBE — A Bayesian Back End for Design of Experiments
-
-This software package provides a general-purpose toolbox for **Design of Experiments
-(DOE)**.
+<div align="center">
+  <br/>
 
-It provides the necessary functionality to:
+[![CI](https://img.shields.io/github/actions/workflow/status/emdgroup/BayBE/ci.yml?style=flat-square&label=CI&labelColor=0f69af&color=ffdcb9)](https://github.com/emdgroup/BayBE/actions?query=workflow%3Aci.yml)
+[![Regular](https://img.shields.io/github/actions/workflow/status/emdgroup/BayBE/regular.yml?style=flat-square&label=Regular%20Check&labelColor=0f69af&color=ffdcb9)](https://github.com/emdgroup/BayBE/actions?query=workflow%3Aregular.yml)
+[![Docs](https://img.shields.io/github/actions/workflow/status/emdgroup/BayBE/docs.yml?style=flat-square&label=Docs&labelColor=0f69af&color=ffdcb9)](https://github.com/emdgroup/BayBE/actions?query=workflow%3Adocs.yml)
 
-- manage experimental design spaces that can be defined through various types of
-  experimental parameters.
-- execute different kinds of DOE strategies, levering both classical and
-  machine-learning-based models.
-- handle measurements data and feed it back into the experimental design.
-- compare different DOE strategies through backtesting with synthetic and real data.
+[![Supported Python version](https://img.shields.io/pypi/pyversions/baybe?style=flat-square&label=Supported%20Python%20Versions&labelColor=96d7d2&color=ffdcb9)](https://pypi.org/project/baybe/)
+[![PyPI version](https://img.shields.io/pypi/v/baybe.svg?style=flat-square&label=PyPI%20Version&labelColor=96d7d2&color=ffdcb9)](https://pypi.org/project/baybe/)
+[![Conda version](https://img.shields.io/conda/vn/conda-forge/baybe.svg?style=flat-square&label=Conda%20Forge%20Version&labelColor=96d7d2&color=ffdcb9)](https://anaconda.org/conda-forge/baybe)
+[![Issues](https://img.shields.io/github/issues/emdgroup/BayBE?style=flat-square&label=Issues&labelColor=96d7d2&color=ffdcb9)](https://github.com/emdgroup/BayBE/issues)
+[![PRs](https://img.shields.io/github/issues-pr/emdgroup/BayBE?style=flat-square&label=PRs&labelColor=96d7d2&color=ffdcb9)](https://github.com/emdgroup/BayBE/pulls)
+[![License](https://shields.io/badge/License-Apache%202.0-green.svg?style=flat-square&labelColor=96d7d2&color=ffdcb9)](http://www.apache.org/licenses/LICENSE-2.0)
 
-## ❗🚧 This repository is under **heavy active development**. 🚧❗
+[![Logo](https://raw.githubusercontent.com/emdgroup/BayBE/main/docs/_static/banner2.svg)](https://github.com/emdgroup/BayBE)

```

Comment:
Ok, as long as it is acceptable that the tox docs test will then fail, this is fine (same with the other one). I'll let you resolve in case this does not cause problems.

Reply:
i dont think we can expect a fully valid docs build at this stage, so imo that pipe needs to irgnore errors for the moment

Reply:
Ok, I think that is acceptable (and probably even necessary). Resolving.